data/phi1348/abo013/phi1348.abo013.perseus-lat2.xml

```diff
@@ -48,1123 +67,1179 @@
-          sibi deficienti anulum mox resipiscens requisisset. Seneca eum scribit intellecta
-          defectione exemptum anulum quasi alicui traditurum parumper tenuisse, dein rursus aptasse
-          digito et compressa sinistra manu iacuisse diu immobilem; subito uocatis ministris ac
-          nemine respondente consurrexisse nec procul a lectulo deficientibus uiribus
-          concidisse.</p>
-      </div>
-      <div type="textpart" n="74" subtype="chapter">
-        <milestone n="1" unit="section"/>
-        <p>Supremo natali suo Apollinem Temenitem et amplitudinis et artis eximiae, aduectum
-          Syracusis ut in bibliotheca templi noui poneretur, uiderat per quietem affirmantem sibi
-          non posse se ab ipso dedicari. et ante paucos quam obiret dies, turris Phari terrae motu
-          Capreis concidit. ac Miseni cinis e fauilla et carbonibus ad calficiendum triclinium
-          inlatis, extinctus iam et diu frigidus, exarsit repente prima uespera atque in multam
-          noctem pertinaciter luxit.</p>
-      </div>
-      <div type="textpart" n="75" subtype="chapter">
-        <milestone n="1" unit="section"/>
-        <p>Morte eius ita laetatus est populus, ut ad primum nuntium discurrentes pars: 'Tiberium in
-          Tiberim!' clamitarent, pars Terram matrem deosque Manes orarent, ne mortuo sedem ullam
-          nisi inter impios darent, alii uncum et Gemonias cadaueri minarentur, exacerbati super
-          memoriam pristinae crudelitatis etiam recenti atrocitate. <milestone n="2" unit="section"/> nam cum senatus consulto cautum esset, ut poena damnatorum in decimum semper diem
-          differretur, forte accidit ut quorundam supplicii dies is esset, quo nuntiatum de Tiberio
-          erat. hos implorantis hominum fidem, quia absente adhuc Gaio nemo extabat qui adiri
-          interpellarique posset, custodes, ne quid aduersus constitutum facerent, strangulauerunt
-          abieceruntque in Gemonias. <milestone n="3" unit="section"/> creuit igitur inuidia, quasi
-          etiam post mortem tyranni saeuitia permanente. corpus ut moueri a Miseno coepit,
-          conclamantibus plerisque Atellam potius deferendum et in amphitheatro semiustilandum,
-          Romam per milites deportatum est crematumque publico funere.</p>
-      </div>
-      <div type="textpart" n="76" subtype="chapter">
-        <milestone n="1" unit="section"/>
-        <p>Testamentum duplex ante biennium fecerat, alterum sua, alterum liberti manu, sed eodem
-          exemplo, obsignaueratque etiam humillimorum signis. eo testamento heredes aequis partibus
-          reliquit Gaium ex Germanico et Tiberium ex Druso nepotes substituitque in uicem; dedit et
-          legata plerisque, inter quos uirginibus Vestalibus, sed et militibus uniuersis plebeique
-          Romanae uiritim atque etiam separatim uicorum magistris. <milestone n="4" unit="Book"/></p>
-      </div>
-    </div></body>
+    <body>
```

Comment:
looks like the reformatting changed the entire file here....

Reply:
I did a pretty print to format the bibliographic information, because I had used a find and replace across the files to copy it in (which made it all appear on one long  line). There were no actual textual changes made or structural changes to the file itself, I had forgotten that it would show the entire file as changing.